src/bowling.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+class Bowling {
+
+  constructor( game = new Game) {
+    this.game = game;
+    this.scoreArray = [];
+  };
+
+  score(scoreArray) {
+    this.scoreArray = scoreArray
+    this.scoresToFrames()
+    this.addSpareBonuses()
+    this.addStrikeBonuses()
+    return this.accumulator()
+  };
+
+  scoresToFrames() {
```

Comment:
Are there any methods that could be made private? In this class or other classes